qa/rpc-tests/excessive.py

```diff
@@ -12,17 +12,231 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal
 from test_framework.util import *
+from test_framework.blocktools import *
+import test_framework.script as script
 import pdb
+import sys
+import logging
+logging.basicConfig(format='%(asctime)s.%(levelname)s: %(message)s', level=logging.INFO)
 
 class ExcessiveBlockTest (BitcoinTestFramework):
+    def __init__(self,extended=False):
+      self.extended = extended
+      BitcoinTestFramework.__init__(self)
 
-    def run_test (self):
-        BitcoinTestFramework.run_test (self)
+    def setup_network(self, split=False):
+        self.nodes = start_nodes(4, self.options.tmpdir,timewait=60*10)
```

Comment:
it's possible to override Travis wait time ... (just saying, it doesn't seem much use if total time is limited to 50min)